README.md

```diff
@@ -27,20 +27,20 @@ This will create:
 
 ### Role variables
 
-* `name` (**required**) - name of the service
+* `name` (**required**) - name of the container
 
 #### Docker container specifics
 
-* `image` (**required**) - Docker image the service uses
-* `args` - arbitrary list of arguments to the `docker run` command
-* `cmd` - optional command to the container run command (the part after the
+* `container_image` (**required**) - Docker image the service uses
+* `container_args` - arbitrary list of arguments to the `docker run` command
+* `container_cmd` - optional command to the container run command (the part after the
   image name)
-* `env` - key/value pairs of ENV vars that need to be present
-* `volumes` (default: _[]_) - List of `-v` arguments
-* `ports` (default: _[]_) - List of `-p` arguments
-* `link` (default: _[]_) - List of `--link` arguments
-* `labels` (default: _[]_) - List of `-l` arguments
-* `docker_pull` (default: _yes_) - whether the docker image should be pulled
+* `container_env` - key/value pairs of ENV vars that need to be present
```

Comment:
In the template `env.j2`, you don't check whether `container_env` is set. I noticed this because I forgot to rename `env` to `container_env`.